CHANGELOG.md

```diff
@@ -41,6 +41,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 * (x/gov) [#14347](https://github.com/cosmos/cosmos-sdk/pull/14347) Support `v1.Proposal` message in `v1beta1.Proposal.Content`.
 
+### API Breaking
+
+* (x/gov) Remove `Migrate_V046_6_To_V046_7` function which shouldn't be used for chains which already migrated to 0.46.
```

Comment:
Don't think it warrants a changelog, actually, because people on 0.46.7 won't see it anyway.

Reply:
I don't mind removing it, but technically it's a diff between 0.46.7 and 0.46.8. Any other opinion?

Reply:
makes sense.